content/en/install.md

```diff
@@ -3,101 +3,149 @@ title: Installation
+a Conda package, so that you can use the same workflow for packages which
+are not available on PyPI.
 
-<a name="conda-install"></a>
+You can add [SciPy from conda-forge] to a [`pixi`] project:
 
-# Installing via Conda
+    pixi add scipy
 
-You can install SciPy from the `defaults` or `conda-forge` channels with `conda`:
+Pixi can also install packages from PyPI, using `uv` under the hood.
 
-    conda install scipy
+[`pixi`]: https://pixi.sh/latest/
 
-<a name="package_manager"></a>
+<a name="system-package-managers"></a>
 
-# Install system-wide via a package manager
+## Installing system-wide via a system package manager
 
 System package managers can install the most common Python packages.
 They install packages for the entire computer, often use older versions,
-and don't have as many available versions.
+and don't have as many available versions. They are not the recommended
+installation method, but they are the simplest.
 
-## Ubuntu and Debian
+### Ubuntu and Debian
 
 Using `apt-get`:
 
     sudo apt-get install python3-scipy
 
-## Fedora
+### Fedora
 
 Using `dnf`:
 
     sudo dnf install python3-scipy
 
-## macOS
+### macOS
 
 macOS doesn't have a preinstalled package manager, but you can install
 [Homebrew](https://brew.sh/) and use it to install SciPy (and Python itself):
 
     brew install scipy
 
-<a name="source"></a>
+<a name="building-from-source"></a>
 
-# Source packages
+## Building from source
 
 A word of warning: building SciPy from source can be a nontrivial exercise. We
-recommend using binaries instead if those are available for your platform.
+recommend using binaries instead if those are available for your platform
+via one of the above methods.
 For details on how to build from source, see
-[this guide in the SciPy docs](https://scipy.github.io/devdocs/building/index.html).
+[the building from source guide in the SciPy docs][building-docs].
+
+[building-docs]: https://scipy.github.io/devdocs/building/index.html
+
+<a name="type-stubs"></a>
+
+## Installing with type stubs
+
+Static type stubs are available via a separate package, `scipy-stubs`, on
+PyPI and conda-forge. If you have already installed SciPy, you just need to
+install `scipy-stubs` in the same way:
+
+    python -m pip install scipy-stubs # or
+    uv add scipy-stubs # or
+    conda install scipy-stubs # or
+    pixi add scipy-stubs
+
+<!---
+XXX: https://github.com/conda-forge/scipy-stubs-feedstock/pull/5
+-->
+
+You can also install SciPy and `scipy-stubs` as a single package,
+via the `scipy-stubs[scipy]` extra on PyPI, or the `scipy-typed`
+package on conda-forge.
+To get a specific version `x.y.z` of SciPy (such as `1.14.1`),
+you should install version `x.y.z.*` below:
+
+    python -m pip install "scipy-stubs[scipy]" # or
+    # versions for illustrative purposes
+    uv add "scipy-stubs[scipy]==1.14.1.*" # or
+    conda install "scipy-typed>=1.14" # or
+    pixi add "scipy-typed=1.15.0.*"
```

Comment:
```suggestion
```

I suggest keeping this section short, to keep the overall document concise. These instructions seem fairly self-evident, and is likely also documented on the pypi / conda pages for `scipy-stubs`.

Reply:
Can we combine the code snippets, perhaps, so there's One Recommended Way of installing SciPy + stubs?

Reply:
> One Recommended Way of installing SciPy + stubs?

On the one hand, most existing users will just want to now how to install the stubs on top of their existing SciPy installation. But on the other hand, installing both at once seems much more suitable for a "how to install SciPy" guide, and ensures versions stay in sync. I don't think I'm confident choosing one over the other.

Reply:
perhaps we just mention how to install both at once here, but also say something like "to install `scipy-stubs` and `scipy` at once". Then users with existing installations should be able to infer that they can just install `scipy-stubs` instead.

Reply:
> I don't think the versioning of scipy-stubs is immediately self-evident without someone being told about it

Most stubs-only packages use a similar versioning scheme, but there are probably many that haven't used stubs-only packages before.